python/lsst/ts/ATDome/mock_controller.py

```diff
@@ -316,14 +328,14 @@ def do_short_status(self):
             dir_code = "RL"
         if az_moving:
             if self.last_rot_right:
-                move_code += 1
+                move_code += MoveCode.AZIMUTH_POSITIVE
```

Comment:
As move_code is a bitmask, more obvious (and also a bit quicker in CPU costs) would be to use |= operator